incubator/cassandra/values.yaml

```diff
@@ -192,10 +192,14 @@ backup:
       memory: 1Gi
       cpu: 1
 
+  # Name of the secret containing the credentials of the service account used by GOOGLE_APPLICATION_CREDENTIALS, as a credentials.json file
+  # google:
+    # serviceAccountSecret:
+
   # Destination to store the backup artifacts
   # Supported cloud storage services: AWS S3, Minio S3, Azure Blob Storage
```

Comment:
Maybe most important - can you add `Google Cloud Storage` to this list? :)

Reply:
Eh indeed!